specification/components/r4/schemas/responses/searchForService/400Response.yaml

```diff
@@ -1,11 +1,10 @@
 description: |
-  Where status code 400 (Bad Request) is returned then an eRS-OperationOutcome-1 will be included in the body, as detailed below. 
+  Where status code 400 (Bad Request) is returned then an [NHSDigital-OperationOutcome](https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome) will be included in the body, as detailed below. 
   Check diagnostics property for specific information regarding the error.
 
-  | Error code               | Description                                                                      |
-  | ------------------------ | -------------------------------------------------------------------------------- |
-  | INVALID_VALUE            | Service Id format is invalid or missing                                          |
-  | INAPPROPRIATE_VALUE      | 'Number of Service Ids' exceeds the range. Up to 20 service IDs can be provided. |
+  | issue.details.coding.code | issue.code | Coding System                                                      | Description                                                                                                                                       |
+  | ------------------------- | ---------- | ------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+  | REC_BAD_REQUEST           | invalid   | [BaRS Error Code](https://fhir.nhs.uk/CodeSystem/http-error-codes) | Incorrect or missing expected value. For example Service Id format is invalid or missing, Another scenario is requesting more than 20 service IDs |
```

Comment:
Can you put the example bit on a new line? Not sure if it lets you?

How about:
"For example; Service Id is invalid or missing, or an an attempt to request more than 20 Service Ids.